packages/server/lib/controllers/auth.controller.ts

```diff
@@ -260,6 +292,79 @@ class AuthController {
             next(error);
         }
     }
+
+    getHostedLogin(req: Request, res: Response, next: NextFunction) {
+        try {
+            const provider = req.query['provider'] as string;
+
+            if (!workos) {
+                errorManager.errRes(res, 'workos_not_configured');
+                return;
+            }
```

Comment:
see my comment above. This is not a concern for the user and I think it would be better to crash the server at startup

Reply:
WorkOS is only required for Cloud so don't think it should crash here.

Reply:
what I mean is this is a configuration issue that would better detected as soon as possible (aka: when the app starts) rather than deferred to whenever a request is being received. 
For sure for non-cloud setup this configuration can be ignored

Reply:
Yup, I gotcha, but who are we protecting here? Developers running the application locally? Cloud will have the necessary env variables.

Reply:
> Yup, I gotcha, but who are we protecting here? 
We protecting our future self

> Cloud will have the necessary env variables.
They will, until one day the infra changes and configuration is somehow forgotten or messed up.
In general trying to detect error as soon as possible is a good principle to follow.

Not a blocker though.